packages/react-components/react-spinner/src/components/Spinner/useSpinnerStyles.ts

```diff
@@ -255,7 +256,7 @@ const useTrackStyles = makeStyles({
 const useLabelStyles = makeStyles({
   // style for label
   inverted: {
-    color: tokens.colorNeutralForegroundStaticInverted,
+    color: 'rgba(255, 255, 255, 0.2)', // This is white alpha but the token is not exported
```

Comment:
Should the label color have an alpha applied? Won't this make the label nearly invisible?

Reply:
This is for the inverted styling. The `colorNeutralForegroundStaticInverted` token doesn't exist, so the guidance was to use white alpha